frontend/src/app/main/store.cljs

```diff
@@ -46,17 +46,10 @@
          (rx/subs #(reset! buffer (vec %))))
     buffer))
 
-(defn emit!
-  ([] nil)
-  ([event]
-   (ptk/emit! state event)
-   nil)
-  ([event & events]
-   (apply ptk/emit! state (cons event events))
-   nil))
+(def emit! (partial ptk/emit! state))
 
 (defn emitf
   [& events]
-  #(apply ptk/emit! state events))
+  #(ptk/emit! state events))
```

Comment:
This is not the same code, on the new code you executing `ptk/emit!` with state and always a seq of events but this function is variadic one. In fact, this probable breaks some other code that uses emitf.

Reply:
OK, I'll revert this change